testData/generate/Getter1.txt

```diff
@@ -1,7 +1,7 @@
 class Getter1 {
-  var foo(getFoo, null):Int;
+  @:isVar public var foo(get, null):Int;
```

Comment:
I don't think that's part of this task, though.  Just pointing out that we're likely to get bugs in those areas.

BTW, in the plugin, not TiVo code.